src/modules/account/components/account-deposit/account-deposit.jsx

```diff
@@ -92,25 +92,65 @@ export default class AccountDeposit extends Component {
         </div>
         <div className={Styles.AccountDeposit__main}>
           <div className={Styles.AccountDeposit__description}>
-            {show0xInstant && (
+            {
+              <>
+                <p>1. Click on your address on the right to copy it</p>
+                <p>
+                  {
+                    "2. Go to Coinbase or Wyre (below) and buy Ether/ETH, and paste in your address to send it. Note if you want immediate access use a debit card, otherwise you'll have to wait a few days."
+                  }
+                </p>
+                <p>
+                  {
+                    "3. Come back here after and resume whatever you were doing!"
+                  }
+                </p>
+              </>
+            }
+            {
               <div className={Styles.AccountDeposit__0xInstantButton}>
-                <button onClick={openZeroExInstant}>
-                  Buy REP using 0x instant
+                <button onClick={() => window.open("http://www.sendwyre.com/")}>
+                  Buy ETH (for trading) using Wyre
                 </button>
               </div>
+            }
+            {
+              <>
+                <br />
+                <div className={Styles.AccountDeposit__0xInstantButton}>
+                  <button
+                    onClick={() => window.open("http://www.coinbase.com/")}
+                  >
+                    Buy ETH (for trading) using Coinbase
+                  </button>
+                </div>
+              </>
+            }
+            {show0xInstant && (
+              <>
+                <br />
+                <div className={Styles.AccountDeposit__0xInstantButton}>
+                  <button onClick={openZeroExInstant}>
+                    Buy REP (for reporting) using 0x instant
+                  </button>
+                </div>
+              </>
             )}
```

Comment:
I think we should add more spacing or have some sort of divider between the buttons for buying ETH and buying REP. This will look a bit strange on mainnet/Koven with all the buttons squished together:

<img width="407" alt="screen shot 2019-01-02 at 4 02 07 pm" src="https://user-images.githubusercontent.com/11491365/50618498-f7216900-0ea7-11e9-9145-80ad95db2217.png">

Reply:
Fixed